fmriprep/workflows/bold/resampling.py

```diff
@@ -167,128 +168,156 @@ def select_target(subject_id, space):
+            or paths pointing to custom templates organized in a TemplateFlow-like structure.
+            Values of the dictionary aggregate modifiers (e.g. the value for the key ``MNI152Lin``
+            could be ``{'resolution': 2}`` if one wants the resampling to be done on the 2mm
+            resolution version of the selected template).
         mem_gb : float
             Size of BOLD file in GB
         omp_nthreads : int
             Maximum number of threads an individual process may use
         name : str
-            Name of workflow (default: ``bold_mni_trans_wf``)
-        template_out_grid : str
-            Keyword ('native', '1mm' or '2mm') or path of custom reference
-            image for normalization.
+            Name of workflow (default: ``bold_std_trans_wf``)
         use_compression : bool
             Save registered BOLD series as ``.nii.gz``
         use_fieldwarp : bool
             Include SDC warp in single-shot transform from BOLD to MNI
 
     **Inputs**
 
-        itk_bold_to_t1
-            Affine transform from ``ref_bold_brain`` to T1 space (ITK format)
-        t1_2_mni_forward_transform
-            ANTs-compatible affine-and-warp transform file
-        bold_split
-            Individual 3D volumes, not motion corrected
-        bold_mask
-            Skull-stripping mask of reference image
-        bold_aseg
-            FreeSurfer's ``aseg.mgz`` atlas projected into the T1w reference
-            (only if ``recon-all`` was run).
+        anat2std_xfm
+            List of anatomical-to-standard space transforms generated during
+            spatial normalization.
         bold_aparc
             FreeSurfer's ``aparc+aseg.mgz`` atlas projected into the T1w reference
             (only if ``recon-all`` was run).
+        bold_aseg
+            FreeSurfer's ``aseg.mgz`` atlas projected into the T1w reference
+            (only if ``recon-all`` was run).
+        bold_mask
+            Skull-stripping mask of reference image
+        bold_split
+            Individual 3D volumes, not motion corrected
+        fieldwarp
+            a :abbr:`DFM (displacements field map)` in ITK format
+        hmc_xforms
+            List of affine transforms aligning each volume to ``ref_image`` in ITK format
+        itk_bold_to_t1
+            Affine transform from ``ref_bold_brain`` to T1 space (ITK format)
         name_source
             BOLD series NIfTI file
             Used to recover original information lost during processing
-        hmc_xforms
-            List of affine transforms aligning each volume to ``ref_image`` in ITK format
-        fieldwarp
-            a :abbr:`DFM (displacements field map)` in ITK format
+        templates
+            List of templates that were applied as targets during
+            spatial normalization.
 
-    **Outputs**
+    **Outputs** - Two outputnodes are available. One output node will be parameterized
```

Comment:
I don't think anybody will know what "parameterized" means.

Reply:
Will revise in the `docs/` branch